src/Sarif.FunctionalTests/SarifConverterTestData/AndroidStudio/AndroidStudio_issueLog1_raw.xml.sarif

```diff
@@ -17,6 +17,36 @@
           }
         ]
       },
+      "logicalLocations": {
+        "app\\microsoft.androidstudioformatexample\\microsoft.androidstudioformatexample.ApplicationTest ApplicationTest()": [
+          {
+            "name": "app",
+            "kind": "android-module"
```

Comment:
I didn't produce it. That's what the converter was producing and I didn't change it. (See the left side of the diff, below.) And it produces it because we do indeed have an enumeration value `LogicalLocationKind.AndroidModule`. Do you want to remove it or add it to the spec and the schema?